### ISSUE TITLE
Missing methods in npm version

### ISSUE THREAD

**Issue by ghost** [giangvo/alfred-workflow-nodejs#3]
Hey,

I'm currently trying to get my first alfred worflow with your nice framework up and running. I noticed that the npm package is slightly different than the github version. Especially this method of the ``Actionhandler`` is totally missing:

```
        onMenuItemSelected: function(action, handler) {
            if (!action || !handler) {
                return;
            }
            eventEmitter.on("menuItemSelected-" + action, handler);
        },
```

**Comment by giangvo** [comment]
yup, the code in master is not a released version. I am developing menu system for Alfred ( I should do it on branch but I push it into master by mistake). If you are using released version, you can checkout the tag. Sorry for that inconvenience.

**Comment by giangvo** [comment]
Hi,
I've released version 1.0.2 with some bug fixing and remove all code related to menu system. So that the code in npm is same as master branch.

Enjoy!

**Comment by zusatzstoff** [comment]
Wow that was fast :+1: I solved my problem and will ramp the workflow up in a few minutes :)

**Comment by giangvo** [comment]
I've released a new version (1.1.0) which has menu system. It is cool. Let's have a try :)
